src/Test/PdbUtilities/Writer/SymWriterTestUtilities.cs

```diff
@@ -13,5 +13,10 @@ internal static class SymWriterTestUtilities
     {
         public static readonly Func<ISymWriterMetadataProvider, SymUnmanagedWriter> ThrowingFactory =
             _ => throw new SymUnmanagedWriterException("xxx", new NotSupportedException(), "<lib name>");
+
+        public static SymUnmanagedWriter CreateUnmanagedWriter(ISymWriterMetadataProvider metadataProvider)
```

Comment:
What's the benefit of this factory method vs. using `SymUnmanagedWriterFactory.CreateWriter` directly?

Reply:
SymUnmanagedWriterFactory when used from PDBTests raises a compile-time error because the type is defined both in Microsoft.DiaSymReader and Microsoft.CodeAnalysis assemblies. This utility project only references Microsoft.CodeAnalysis, so this factory method was a simple way to workaround that problem (without the need for extern aliases).